algolia/opt/disable_standard_entries.go

```diff
@@ -0,0 +1,63 @@
+// Code generated by go generate. DO NOT EDIT.
+
+package opt
+
+import (
+	"encoding/json"
+	"reflect"
+)
+
+// DisableStandardEntriesOption is a wrapper for an DisableStandardEntries option parameter. It holds
+// the actual value of the option that can be accessed by calling Get.
+type DisableStandardEntriesOption struct {
```

Comment:
The DisableStandardEntriesOption is contextual to Dictionary, but this is lost in the name/package/code comment. Any way to change this?

Reply:
This is a generated code as for all the other properties. I don't think we can group generated properties better than keep them in the `opt` package .